packages/rainbowkit/src/components/ChainModal/ChainModal.test.tsx

```diff
@@ -86,65 +71,20 @@ describe('<ChainModal />', () => {
     expect(onCloseGotCalled).toBe(true);
   });
 
-  it('Just closes on switch error (user rejected, or other)', async () => {
-    let onCloseGotCalled = false;
-    const modal = renderWithProviders(
-      <ChainModal onClose={() => (onCloseGotCalled = true)} open />,
-      {
-        mock: true,
-        mockOptions: {
-          chainId: mainnet.id,
-          flags: { failSwitchChain: true, isAuthorized: true },
-        },
-      },
-    );
```

Comment:
Had to remove this test since wagmi doesn't provide any failed switch chain flags. Here are more info on their [mock features](https://wagmi.sh/core/api/connectors/mock#features)